### PR TITLE
moved index2.html to /v2/index.html

### PR DESCRIPTION
**Pull request by Forchapeatl**
moved index2.html to /v2/index.html

Fixes #0000 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by Forchapeatl** [comment]
@jywarren , please how may I relate the path of these files after moving `index2.html` to `v2/index.html` ?
image

**Comment by jywarren** [comment]
Hi! Take a look at this example!

https://github.com/publiclab/infragram/blob/Forchapeatl-patch-13/vr/index.html#L19

**Comment by jywarren** [comment]
Ok I think you need to adjust the test paths too now!

**Comment by jywarren** [comment]
Good work though!!

**Comment by Forchapeatl** [comment]
Thank you @jywarren ,Please what should I do about the bootstrap problem? When I includes it shader files(WebGL) do not lot. If I exclude it the UI will get destroyed.

I was thinking  of including the shader*  files in our v2 directory.

.
├── ...
├── v2                  
│   ├── index.html
│   │── dist         
│   │──── shader.frag
│   └──── shader.vert       
└── ...

**Comment by jywarren** [comment]
Ah, sorry i missed that question. See how in these examples, we made the path to those files settable using configuration options? https://github.com/publiclab/infragram/blob/5ff8f3fce06410c7729a77fc00e0cc9668462554/vr/index.html#L184-L185

That should work! Yes, we encountered this same issue before for those versions!

**Comment by Forchapeatl** [comment]
Thank you @jywarren . May I have permission to merge ?

File: v2/index.html [hunk 1]
```diff
@@ -509,7 +509,9 @@ <h6 style="text-decoration:underline;">BLUE filters <i class="fa fa-filter" aria
         infragram = Infragram({
           uploader: false,
           processor: 'webgl',
-          version: 2
+          version: 2,
+          shaderVertPath: "../dist/shader.vert",
```
Comment (jywarren) [inline v2/index.html:513]: Perfect!

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Great. Let's be sure @stephaniequintana knows we've moved the file!

**Comment by jywarren** [comment]
I approved so go ahead!!

**Pull request merged**